library/globals.inc.php

```diff
@@ -2635,7 +2629,7 @@
         'ms' => xl('Malay'),
         'ml' => xl('Malayalam'),
         'mt' => xl('Maltese'),
-        'mi' => xl('Māori'),
+        'mi' => xl('Maori'),
         'mr' => xl('Marathi (Marāṭhī)'),
```

Comment:
If you are removing non-US keyboard characters, maybe you should do all of them before they get translated here?  I think this stuff should only be included when translations are on (period).